airflow/example_dags/example_skip_dag.py

```diff
@@ -37,23 +33,15 @@ def execute(self, context):
         raise AirflowSkipException
 
 
-dag = DAG(dag_id='example_skip_dag', default_args=args)
-
-
 def create_test_pipeline(suffix, trigger_rule, dag):
-
     skip_operator = DummySkipOperator(task_id='skip_operator_{}'.format(suffix), dag=dag)
-
     always_true = DummyOperator(task_id='always_true_{}'.format(suffix), dag=dag)
-
     join = DummyOperator(task_id=trigger_rule, dag=dag, trigger_rule=trigger_rule)
-
-    join.set_upstream(skip_operator)
-    join.set_upstream(always_true)
-
     final = DummyOperator(task_id='final_{}'.format(suffix), dag=dag)
-    final.set_upstream(join)
+
+    [skip_operator, always_true] >> join >> final
```

Comment:
Is this allowed?

Reply:
Whoops my bad. Checked the `BaseOperator.__rshift__` and that accepts a single task or List of tasks for the `other` object. But obviously calling rshift on a Python List doesn't work :-) Will fix.